docroot/sites/all/modules/patched/privatemsg/patches/privatemsg.pm_email_notify_variable_based_localized_email_2505495_01.patch

```diff
@@ -0,0 +1,45 @@
+commit 71718b798ab0cec1ca464d33835d053f148b3b5f
+Author: Randy Fay <randy@randyfay.com>
+Date:   Fri Jun 12 17:59:36 2015 -0600
+
+    Issue #2505495: Add standard variable-based localized email sending to pm_email_notify
+
+diff --git a/pm_email_notify/pm_email_notify.module b/pm_email_notify/pm_email_notify.module
+index 065dfd4..b7a01df 100644
+--- a/pm_email_notify/pm_email_notify.module
++++ b/pm_email_notify/pm_email_notify.module
+@@ -50,6 +50,7 @@ function pm_email_notify_privatemsg_message_insert($message) {
+         'privatemsg-display-invalid' => FALSE,
+       );
+       $from = trim(token_replace(variable_get('pm_email_notify_from', ''), $data, $options));
++
+       drupal_mail('pm_email_notify', 'notice', $recipient->mail, user_preferred_language($recipient), $params, !empty($from) ? $from : NULL);
+     }
+   }
+@@ -65,6 +66,14 @@ function pm_email_notify_mail($key, &$message, $params) {
+         'privatemsg_message' => $params['message'],
+         'privatemsg_recipient' => $params['recipient'],
+       );
++
++      $subject = variable_get('pm_email_notify_subject', 'New private message at [site:name].');
++      $body = variable_get('pm_email_notify_body', 'New private message at [site:name].');
++      if (function_exists('i18n_variable_get')) {
++        $subject = i18n_variable_get('pm_email_notify_subject', user_preferred_language($params['recipient'])->language, 'New private message at [site:name].');
++        $body = i18n_variable_get('pm_email_notify_body', user_preferred_language($params['recipient'])->language, _pm_email_notify_default_body());
++      }
```

Comment:
I've used this modality for decades now, and it is ever so slightly less efficient in some cases, but it accomplishes three things. 1) Makes sure variable is initialized, 2) Shows the "normal" or most common flow in the code, 3) Avoids unnecessary and hard-to-follow "else".  So I like it. It's a tiny bit less efficient, but in this context performance is completely irrelevant.